src/PlusDataCollection/IntersonArraySDKCxx/vtkPlusIntersonArraySDKCxxVideoSource.cxx

```diff
@@ -316,21 +314,30 @@ PlusStatus vtkPlusIntersonArraySDKCxxVideoSource::InternalConnect()
   vtkPlusDataSource * source = NULL;
   this->GetVideoSourcesByPortName(vtkPlusDevice::RFMODE_PORT_NAME, rfSources);
 
-  int width;
+  int width_samples;
   if( !rfSources.empty() )
     {
     container->SetRFData( true );
-    width = ContainerType::MAX_RFSAMPLES;
+    width_samples = ContainerType::MAX_RFSAMPLES;
     }
   else
     {
     container->SetRFData( false );
-    width = ContainerType::MAX_SAMPLES/2;
+    width_samples = ContainerType::MAX_SAMPLES/2;
     }
 
   LOG_DEBUG( "Interson Array SDK version " << this->Internal->GetSdkVersion() <<
              ", USB probe FPGA version " << hwControls->ReadFPGAVersion() );
 
+  // Even if we do not use their SDK scan converter, we have to initialize the
+  // scan converter to get the probe fully initialized.
+  const int depth = 209;
+  const int height_lines = hwControls->GetLinesPerArray();
+  const int steering = 0;
+  container->IdleInitScanConverter( depth, width_samples, height_lines, probeId,
+    steering, false, false, 0 );
+  container->HardInitScanConverter( depth, width_samples, height_lines, steering );
+
   std::vector<vtkPlusDataSource *> bmodeSources;
```

Comment:
To be consistent with the rest of PlusLib, please use std::vector<vtkPlusDataSource*> instead of std::vector<vtkPlusDataSource *> (no space before *)